apps/preview/next/package.json

```diff
@@ -32,6 +32,7 @@
     "@storefront-ui/tests-shared": "workspace:*",
     "@storefront-ui/tw-plugin-peer-next": "workspace:*",
     "@storefront-ui/typography": "workspace:*",
+    "@tailwindcss/typography": "^0.5.9",
```

Comment:
question: do we need to install this plugin if we have our own typography plugin ? I'm not sure how it is working